### DISCUSSION TITLE
scalar bar and pythreejs backend

### DISCUSSION THREAD

**Discussion by PBrockmann** [pyvista/pyvista#2560] [Q&A] [answered]
It seems that when you choose pythreejs backend the rendering of the scalar bar is not done.

Am I missing something ?

```
sphere = pv.Sphere()
sphere['Data'] = sphere.points[:, 2]
plotter = pv.Plotter()
plotter.add_mesh(sphere, show_scalar_bar=True)
plotter.show(jupyter_backend='pythreejs')   # scalar_bar is not visible
```

**Comment by akaszynski** [comment, marked as answer]
@PBrockmann, scalar bars are not yet available in the pythreejs backend plot. We're working on adding that feature, but it might take some time.
